charts/fybrik/templates/_helpers.tpl

```diff
@@ -147,3 +147,14 @@ Relevant only when .Values.chartsPersistentVolumeClaim is set.
 {{- define "fybrik.localChartsMountPath" }}
 {{- printf "/opt/fybrik" }}
 {{- end }}
+
+{{/*
```

Comment:
Why do you need all this?
you can define separatly `useTLS` and optionally port. For http port can be redefine too

Reply:
It is used in the service to check if tls is used